public/assets/js/account.js

```diff
@@ -19,4 +19,44 @@ document.getElementById('remove-discord-connection')?.addEventListener('click',
 			}
 		})
 		.catch(console.log);
+});
+
+const onlineFilesModal = document.querySelector('.online-files-modal-wrapper');
+const onlineFilesModalButtonConfirm = document.getElementById('onlineFilesConfirmButton');
+const onlineFilesModalButtonClose = document.getElementById('onlineFilesCloseButton');
+const onlineFilesModalPasswordInput = document.getElementById('password');
+
+document.getElementById('download-cemu-files')?.addEventListener('click', event => {
+	event.preventDefault();
+
+	onlineFilesModal.classList.remove('hidden');
+});
+
+onlineFilesModalButtonConfirm?.addEventListener('click', () => {
+	fetch('/account/online-files', {
+		method: 'POST',
+		headers: {
+			'Accept': 'application/json',
+			'Content-Type': 'application/json'
+		},
+		body: JSON.stringify({ 
+			password: onlineFilesModalPasswordInput.value
+		})
+	})
+		.then(response => response.blob())
+		.then()
```

Comment:
Is this stray await supposed to be here?

Reply:
welp, that selected more lines it should have. it's 47

Reply:
Fixed